product_tier_validation/tests/test_product_tier_validation.py

```diff
@@ -37,4 +37,4 @@ def test_validation_product_template(self):
         product.request_validation()
         product.with_user(self.test_user_1).validate_tier()
         product.write({"product_state_id": self.normal_state.id})
-        self.assertEqual(product.state, self.normal_state.code)
```

Comment:
Is this test result changed ?

Reply:
Not sure I understand your question. I changed this since in this module instead of setting the state from code, we set it from the state field on the stage therefore the test needed to be updated too in order to be correct.